Source/WebKit/WebProcess/WebPage/Cocoa/TextAnimationController.mm

```diff
@@ -310,18 +332,22 @@
     else {
         auto animationRange = contextRangeForTextAnimationID(uuid);
         if (!animationRange) {
-            completionHandler();
+            if (completionHandler)
+                completionHandler();
             return;
         }
 
         document->markers().addTransparentContentMarker(*animationRange, uuid);
     }
 
-    completionHandler();
+    if (completionHandler)
+        completionHandler();
 }
 
 std::optional<WebCore::TextIndicatorData> TextAnimationController::createTextIndicatorForRange(const WebCore::SimpleRange& range)
 {
+    auto rangeForIndicator = range;
```

Comment:
Why the copy?

Reply:
Oh, this is leftover from when I was trying to grab the snapshot of the placeholder here.